### DISCUSSION TITLE
Welcome to dsccodecollab.github.io Discussions!

### DISCUSSION THREAD

**Discussion by abhinavsri360** [dsccodecollab/dsccodecollab.github.io#26] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by soyo-kaze** [comment]
Once I've pushed and merged some code will it be built for production automatically?

**Reply by soyo-kaze** [reply]
got it. It happened automatically CI works dope have to learn this stuff 🤩

**Reply by abhinavsri360** [reply]
We are using netlify, that's why it happened automatically. There are other ways as well :p
